packages/katello/katello/katello-remove

```diff
@@ -42,8 +43,8 @@ LOG_FILES=[
   '/var/log/tomcat6',
   '/var/log/pulp',
   '/var/log/candlepin',
-  '/var/log/httpd',
   '/var/log/mongodb',
+  '/var/log/httpd',
```

Comment:
It's nice to minimize the amount of lines changed. In this case it doesn't make a difference so I'd change it back to the way it was.